src/utils/url.ts

```diff
@@ -4,6 +4,10 @@ export function wrapUtm(url: string) {
   return `${url}?utm_medium=email&utm_source=email`;
 }
 
+export function wrapCrispUtm(url: string) {
+  return `${url}&utm_medium=email&utm_source=email`;
+}
+
```

Comment:
think should combine this function into `wrapUtm()` and `<CrispLink/>` into `<Link/>` to provide option as second param/props to choose `?` or `&`

Reply:
gotcha! Changed in next commit